### PR TITLE
lib Animation api

### PR DESCRIPTION
**Pull request by chrisdugne**
**Detailed purpose of the PR**

- simplifies the use of `Animated` with a `useAnimation` hook allowing declarative sequences
(requires static callbacks https://github.com/CoorpAcademy/components/compare/rewriting-animated-with-onfinished#diff-ed85277a0ec0a46f22a086ad56fa9944e3403d2fd26af5aa1baa527532c76773R31)

### CODE REVIEW

File: packages/@coorpacademy-components/src/organism/review-congrats/index.native.tsx [hunk 1]
```diff
@@ -68,9 +99,62 @@ const ReviewCongrats = (props: ReviewCongratsProps) => {
...
+          }
+        ]
+      },
+      {
+        type: 'parallel',
+        steps: [fadeInStars, translateStars]
+      },
+      showButtons
+    ]
+  });
```
Comment (chrisdugne) [inline packages/@coorpacademy-components/src/organism/review-congrats/index.native.tsx:148]: objectif

Comment (chrisdugne) [reply packages/@coorpacademy-components/src/organism/review-congrats/index.native.tsx:148]: @godu 's suggestion
```
useAnimation(
  sequence([
    showCongrats,
    parallel([
      showConfettis,
      parallel([
        fadeInRank,
        translateRank
      ])
    ]),
    parallel([
      fadeInStars,
      translateStars
    ]),
    showButtons
  ])
);
```

**Comment by chrisdugne** [comment]
[[OR] Congrats - mobile](https://trello.com/c/E6WcHScz/2571-or-congrats-mobile)

File: packages/@coorpacademy-components/src/organism/review-congrats/index.native.tsx (hunk 1 above)
Comment (chrisdugne) [reply packages/@coorpacademy-components/src/organism/review-congrats/index.native.tsx]: j'ai meme obtenu:
```
  const animation = sequence([
    showCongrats,
    parallel([showConfettis, fadeInRank, translateRank]),
    parallel([fadeInStars, translateStars]),
    showButtons
  ]);
```
https://github.com/CoorpAcademy/components/pull/2524/commits/f393b6b6d72fcffd5ada3577edd007bc83e5a422#diff-3542e65f5c1d97dbb9e5a2c66e7596f08b7f77260d6931d445c97e8cc423a89fR128-R133

**Review by scalabw — APPROVED** [review]

**Comment by chrisdugne** [comment]
https://github.com/CoorpAcademy/components/pull/2530

**Pull request closed without merging**